cmd/cancel.go

```diff
@@ -45,7 +46,12 @@ func newCancelCmd() *cobra.Command {
 			if len(args) > 0 {
 				stack = args[0]
 			}
-			s, err := requireStack(stack, false)
+
+			opts := backend.DisplayOptions{
+				Color: cmdutil.GetGlobalColorization(),
+			}
```

Comment:
For command that do not support a --color flag, this is the pattern to use.  The color picked here will be "Never" if NO_COLOR is set in the environment.  Or 'Always' otherwise.  In the future we can consider if these command shoudl support --color as well.  But i didn't want to have to add that here.